### PR TITLE
Removing add on support

### PR DESCRIPTION
**Pull request by jgzamora**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,11 +82,9 @@ Content-Type: application/json
 }
 ```
 
-### /api/migrate: Migration From Mandrill To SparkPost
-
-Extract a template from Mandrill, translate it and import it into SparkPost.
+#### /api/migrate: Migration From Mandrill To SparkPost
 
-Note: your SparkPost API key must include Templates Read/Write privileges for use with the `migrate` endpoint and the migration UI.
+Extracts a template from Mandrill, translates it and imports it into SparkPost. Your SparkPost API key must include Templates Read/Write permission in order to add templates to the account. Manage your API keys from the [API Keys page](https://app.sparkpost.com/account/api-keys)([EU](https://app.eu.sparkpost.com/account/api-keys)).
```
Comment (ewandennis) [inline README.md]: We don't support SPC EU here so I suggest either we add a control for that or we remove this reference and add a caveat.

File: routes/migrate.js [hunk 2]
```diff
@@ -22,13 +22,7 @@ router.post('/', function(req, res) {
...
-  if (req.body.hasOwnProperty('useHerokuSPAPIKey') && req.body.useHerokuSPAPIKey) {
-    if (process.env.SPARKPOST_API_KEY) {
-      spAPIKey = process.env.SPARKPOST_API_KEY;
-    } else {
-      return res.clientError('Heroku SparkPost API key not found. Are we running under Heroku with the SparkPost addon?');
-    }
-  } else if (!req.body.hasOwnProperty('sparkPostAPIKey')) {
+  if (!req.body.hasOwnProperty('sparkPostAPIKey')) {
     return res.clientError('Expected sparkPostAPIKey field');
   } else {
```
Comment (ewandennis) [inline routes/migrate.js]: We don't need the else clause here.

File: README.md [hunk 3]
```diff
@@ -95,11 +82,9 @@ Content-Type: application/json
 }
 ```
 
-### /api/migrate: Migration From Mandrill To SparkPost
-
-Extract a template from Mandrill, translate it and import it into SparkPost.
+#### /api/migrate: Migration From Mandrill To SparkPost
 
-Note: your SparkPost API key must include Templates Read/Write privileges for use with the `migrate` endpoint and the migration UI.
+Extracts a template from Mandrill, translates it and imports it into SparkPost. Your SparkPost API key must include `Templates Read/Write` permission in order to add templates to your account. Manage your API keys from the [API Keys page](https://app.sparkpost.com/account/api-keys)([EU](https://app.eu.sparkpost.com/account/api-keys)).
```
Comment (ewandennis) [inline README.md]: We don't have EU support. I suggest either we add a control and API field for it or we remove this and add a caveat to the top of README.

**Comment by jgzamora** [comment]
Added EU support and tested migration

**Pull request merged**